hq/app/schedule/IamAudit.scala

```diff
@@ -36,30 +35,38 @@ object IamAudit extends Logging {
     }
   }
 
-  def makeCredentialsNotification(allCreds: Map[AwsAccount, Either[FailedAttempt, CredentialReportDisplay]]): List[Notification] = {
-    allCreds.toList.flatMap { case (awsAccount, eFCreds) =>
-      eFCreds match {
-        case Right(credsReport) =>
-          val outdatedKeys = outdatedKeysInfo(findOldAccessKeys(credsReport))
-          val missingMfa = missingMfaInfo(findMissingMfa(credsReport))
-          val targetGroups = getNotificationTargetGroups(outdatedKeys, missingMfa)
+  def getFlaggedCredentialsReports(allCreds: Map[AwsAccount, Either[FailedAttempt, CredentialReportDisplay]]): Map[AwsAccount, CredentialReportDisplay] = {
```

Comment:
This PR splits up the previous function, `makeCredentialsNotification` into two separate functions. The first, filters credentials that are old and/or missing mfa and and the second, creates a notification for these flagged credentials. This is helpful, because the reaper feature will need to check DynamoDB to see which notifications have already been sent in the past before sending another notification. This will be forthcoming.